src/main/kotlin/com/group/libraryapp/service/user/FailTest.kt

```diff
@@ -4,8 +4,8 @@ class FailTest {
 
 
 
-    fun testtest( a:Int) {}
+    fun testtest22( a:Int) {}
 
-    fun maxLineWarning_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890( a:Int) = Unit
+    fun maxLineWarning_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_1234567890_123456789022( a:Int) = Unit
```

Comment:
⚠️ **[detekt]** <detekt.MaxLineLength> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line detected, which is longer than the defined maximum line length in the code style.